candle/set-theory/Holmakefile

```diff
@@ -1,28 +1,9 @@
-INCLUDES = ../../misc $(HOLDIR)/src/pred_set/src/more_theories
-OPTIONS = QUIT_ON_FAILURE
+INCLUDES = $(HOLDIR)/src/pred_set/src/more_theories $(CAKEMLDIR)/misc
```

Comment:
Don't need to include `more_theories`; these theories are in `sigobj` and always accessible